src/Sarif.Sarifer/FileAndForgetEventName.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.Sarif.Sarifer
+{
+    internal static class FileAndForgetEventName
+    {
+        private const string Prefix = "Microsoft/Sarifer/";
+
+        private static readonly Lazy<string> s_sendDataToViewerFailure = new Lazy<string>(() => MakeEventName("SendDataToViewer/Failure"));
```

Comment:
>s_sendDataToViewerFailure [](start = 45, length = 25)

Can you explain all this complexity to me? Why do we need a lazy initializer? What's the problem with simply creating string constants here? This all looks a little 'LAMC' to me. (whee, look at me code.)

Reply:
Yes, I can do it all with constants.

---
In reply to: [520856684](https://github.com/microsoft/sarif-visualstudio-extension/pull/265#discussion_r520856684) [](ancestors = 520856684)